packages/react/src/Prose/Prose.module.css

```diff
@@ -0,0 +1,205 @@
+/* ---------------------------------------------------------- */
+/* Prose component                                            */
+/* ---------------------------------------------------------- */
+/* Table of contents:                                         */
+/* 1. Container                                               */
+/* 2. Element spacing                                         */
+/* 3. Line-length control                                     */
+/* 4. Headings                                                */
+/* 5. Paragraphs                                              */
+/* 6. Lists                                                   */
+/* 7. Inline elements                                         */
+/* 8. Images                                                  */
```

Comment:
Few other ui elements we may want to consider supporting:

- blockquote
- video / iframe (embeds)
- pre and code

If we don't land these on the first ship, can we please create the issues / add to tasklist for future iterations?

Reply:
As discussed during maintainer's sync yesterday, we'll create follow-up issues to add these elements incrementally.